ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/batch/BatchedSubmissionValidator.scala

```diff
@@ -303,8 +303,8 @@ class BatchedSubmissionValidator[CommitResult] private[validator] (
             )
         }
       }
-      // Commit the results.
-      .mapAsync[Outputs6](params.commitParallelism) {
+      // Commit the results. This must be done serially to ensure a deterministic set of writes.
+      .mapAsync[Outputs6](1) {
```

Comment:
Is this unit-tested anywhere? Perhaps not so important as we are going to have proper (i.e. non-reordering) integrity checks with this one.

Reply:
Yes; the spec file had a test for serial commits which I merged into the other tests.

Reply:
(And it's also tested by the integrity check tests.)